src/tags/filter/gb-filter.ts

```diff
@@ -26,6 +26,8 @@ export class Filter extends SelectTag<FilterOpts> {
     this.expose('selectable');
 
     this.flux.on(FILTER_UPDATED_EVENT, this.updateValues);
+    this.on('unmount', () => this.services.filter.unregister(this));
```

Comment:
I think we should have a register function on the tag itself then it will do both register & unregister that way you dont have to do the unmount or something